app/services/ccms/attribute_value_generator.rb

```diff
@@ -439,7 +425,7 @@ def manual_case_review_required?
     end
 
     def chances_of_success
-      legal_aid_application.application_proceeding_types.first.chances_of_success
+      legal_aid_application.proceedings.first.chances_of_success
```

Comment:
Could you use `lead_proceeding` here instead of `proceedings.first`? Not quite sure what the implications are but I can see you've made that change in other places so it might work here too.